dev/com.ibm.ws.concurrent_fat_reactive/publish/servers/FATServer/server.xml

```diff
@@ -0,0 +1,31 @@
+<!--
+    Copyright (c) 2023 IBM Corporation and others.
+    All rights reserved. This program and the accompanying materials
+    are made available under the terms of the Eclipse Public License 2.0
+    which accompanies this distribution, and is available at
+    http://www.eclipse.org/legal/epl-2.0/
+    
+    SPDX-License-Identifier: EPL-2.0
+   
+    Contributors:
+        IBM Corporation - initial API and implementation
+ -->
+<server>
+
+    <featureManager>
+        <feature>servlet-6.0</feature>
+        <feature>componenttest-2.0</feature>
+        <feature>cdi-4.0</feature>
+        <feature>concurrent-3.0</feature>
+        <feature>jndi-1.0</feature>
+    </featureManager>
+    
+    <include location="../fatTestPorts.xml"/>
+    
+    <managedExecutorService jndiName="concurrent/sampleExecutor">
+        <concurrencyPolicy max="2" maxQueueSize="2"/>
+    </managedExecutorService>
```

Comment:
FYI - I don't see this used anywhere in the tests (searching on `concurrent/sampleExecutor`).

Reply:
Good catch, I pulled that in from another test thinking I would need it. I'll remove it in my next PR